src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -1297,6 +1297,7 @@ public void isNotNullTest() throws IOException {
     }
 
     @Test
+    @Ignore("Use of scroll hint is deprecated in lieu of cursor support")
```

Comment:
if the scrollhint is deprecated should we delete these tests?

Reply:
Even I had the same concern, will delete these tests.